app/src/components/governance/PageProposal.vue

```diff
@@ -170,7 +170,11 @@
       <ModalVoteSubstrate
         v-if="currentNetwork.network_type === `polkadot`"
         ref="modalVote"
-        :proposal-id="proposalId"
+        :proposal-id="
+          currentNetwork.network_type === `polkadot`
+            ? proposal.proposalId
+            : proposalId
```

Comment:
if this is not substrate, we don't need this at all
```suggestion
            : `cosmos-networks-will-not-need-this-component`
```